style_guide.md

```diff
@@ -0,0 +1,270 @@
+# Yield Style Guide
```

Comment:
So a lot of this looks copy/pasted from [solidity style guide](    function testUnit_twar3() public {
        console.log("twar values updated and returned correctly a buy of base tokens");

        // since cumRatLast is on a lag, it should still be zero.
        assertEq(pool.cumulativeRatioLast(), 0);

        // Send FYToken, sync, and calc expected FYTokenIn
        fyToken.mint(address(pool), 5e18);
        uint128 virtFYTokenBal = uint128(fyToken.balanceOf(address(pool)) + pool.totalSupply());
        uint128 sharesReserves = uint128(base.balanceOf(address(pool)));
        int128 c_ = (IERC4626Mock(address(base)).convertToAssets(10 ** base.decimals()).fromUInt()).div(uint256(1e18).fromUInt());

        uint128 expectedFYTokenIn = YieldMath.fyTokenInForSharesOut(
            sharesReserves,
            virtFYTokenBal,
            3e18,
            maturity - uint32(block.timestamp),
            k,
            g2,
            c_,
            mu
        );

        // Send some fyToken to the pool.
        fyToken.mint(address(pool), expectedFYTokenIn); // send an extra wad of base


        // Alice calls mint to Bob.
        vm.startPrank(alice);
        pool.buyBase(alice, 3e18, type(uint128).max);

        // fast forward time
        uint256 timewarp = 100;
        vm.warp(block.timestamp + timewarp);

        // expect the total ratio seconds to be 60 (ray) based on the 1:1 ratio established
        // in setup and the 60 seconds that had elapsed after
        assertEq(pool.cumulativeRatioLast(), 60 * 1e27);

        // expect currCumRat to have increased
        (uint256 currCumRat1,) = pool.currentCumulativeRatio();
        (, uint104 baseReserves, uint104 fyTokenReserves, ) = pool.getCache();
        uint256 expectedCurrCumRat = pool.cumulativeRatioLast() + calcRatioSeconds(fyTokenReserves, baseReserves, timewarp);
        assertEq(currCumRat1, expectedCurrCumRat);

        vm.warp(block.timestamp + 1000);
        expectedCurrCumRat += calcRatioSeconds(fyTokenReserves, baseReserves, 1000);
        (uint256 currCumRat2,) = pool.currentCumulativeRatio();
        assertEq(currCumRat2, expectedCurrCumRat);

    }
), correct?  My initial thought is that we should just link to the styl